### PR TITLE
initial implementation for requirement 2.1

### PR DESCRIPTION
**Pull request by kouissar**
first take on requirement 2.1, please review

### CODE REVIEW

File: css/reveal.css [hunk 1]
```diff
@@ -373,7 +373,7 @@ body {
 .reveal .progress {
   position: absolute;
   display: none;
-  height: 3px;
+  height: 12px;
```
Comment (kemaric) [inline css/reveal.css:376]: For this, look into using rem vs pixels for the size. 
https://www.w3schools.com/cssref/css_units.asp

**Review by OliviaSims21 — APPROVED** [review]
looks good!

**Review by kemaric — APPROVED** [review]

**Comment by kemaric** [comment]
@kouissar you can go ahead and merge this.

**Comment by kouissar** [comment]
Req 2.1 implementation: add progress bar plus page count/total

**Pull request merged**